packages/api/src/schemas/tasks.ts

```diff
@@ -0,0 +1,35 @@
+import { z } from "zod";
+
+export const createTaskCollectionInputSchema = z.object({
+  title: z.string()
+});
+
+export const createTaskInputSchema = z.object({
+  title: z.string().optional(),
+  taskCollectionId: z.string(),
+  description: z.string().optional(),
+  status: z.enum(["needsAction", "completed"]).optional(),
+  accountId: z.string(),
+  completed: z.string().optional(),
+  due: z.string().optional(),
```

Comment:
use `Temporal.PlainDate | Temporal.Instant | Temporal.ZonedDateTime`, 

while google tasks only supports dates this might still be useful for future providers